frame/pablo/src/lib.rs

```diff
@@ -583,16 +555,16 @@ pub mod pallet {
 						min_mint_amount,
 						keep_alive,
 					)?;
-					Self::deposit_event(Event::<T>::LiquidityAdded {
-						who: who.clone(),
-						pool_id,
-						base_amount,
-						quote_amount,
-						minted_lp: T::Balance::zero(),
-					});
+					T::Balance::zero()
```

Comment:
I think it is better to return zero() from LBP's add_liquidity() . It is better to keep logic of LBP hidden from pablo pallet.